scrapy/commands/crawl.py

```diff
@@ -1,9 +1,17 @@
 import os
+import logging
+import sys
 from scrapy.commands import ScrapyCommand
 from scrapy.utils.conf import arglist_to_dict
 from scrapy.utils.python import without_none_values
 from scrapy.exceptions import UsageError
 
+log = logging.getLogger(__name__)
+log.addHandler(logging.StreamHandler())
+log.setLevel(logging.DEBUG)
```

Comment:
Hm, does it respect log level set in scrapy options? Is it necessary to add logger handler and set log level to get output?